inventories/reportinator_server/inventory.yml

```diff
@@ -0,0 +1,22 @@
+---
+reportinator_server:
+  hosts:
+    reportinator.nos.social: 
```

Comment:
I think reportinator.ansible.fun should be listed here too? At least if we want to absorb the `vars` below? Maybe we don't want that.

Reply:
I removed this because of the issue with nos.social subdomains